pgtoolkit/conf.py

```diff
@@ -492,7 +492,12 @@ def parse(self, fo: Iterable[str]) -> Iterator[Tuple[pathlib.Path, IncludeType]]
                 continue
             commented = False
             if line.startswith("#"):
-                # Try to parse the uncommented line.
+                # Try to parse the uncommented line as a commented parameter,
```

Comment:
```suggestion
                # Try to parse the commented line as a commented parameter,
```

It seems like previous doc comment was wrong.

Reply:
Fixed, thanks.